dev/ProgressBar/TestUI/ProgressBarPage.xaml

```diff
@@ -123,7 +123,7 @@
             <TextBlock AutomationProperties.Name="ShowErrorText" Text="{x:Bind ShowErrorCheckBox.IsChecked, Mode=OneWay}"/>
 
             <TextBlock Text="Is indeterminate:" Margin="0, 20, 0, 0" FontSize="20"/>
-            <TextBlock AutomationProperties.Name="ShowIsDeterminateText" Text="{x:Bind ShowIsDeterminateCheckBox.IsChecked, Mode=OneWay}"/>
+            <TextBlock AutomationProperties.Name="ShowIsIndeterminateText" Text="{x:Bind ShowIsIndeterminateCheckBox.IsChecked, Mode=OneWay}"/>
```

Comment:
Same for this change, change the linked line too:

https://github.com/microsoft/microsoft-ui-xaml/blob/1ced221991d3abd51875d24d69221afa878b6c5a/dev/ProgressBar/InteractionTests/ProgressBarTests.cs#L205